### ISSUE TITLE
Any pre-trained models available?

### ISSUE THREAD

**Issue by MinLiAmoy** [HHTseng/video-classification#1]
Sir,

Thanks for your great work. This tutorial is easily readable and excellent!
I wonder are there pre-trained models available? Thanks!

Best regards,
Min

**Comment by HHTseng** [comment]
Hi Min,
Thank you for your nice response and your excellent suggestions. This is a good idea to provide some pretrained models I already have. Please let me know if you can see files from following link: [https://drive.google.com/drive/folders/117mRMS2r09fz4ozkdzN4cExO1sFwcMvs?usp=sharing](https://drive.google.com/drive/folders/117mRMS2r09fz4ozkdzN4cExO1sFwcMvs?usp=sharing)
I will make it more official later

These models were trained on the default parameters provided in this repo. You should be able to run the file, eg. **ResNetCRNN_check_prediction.py**, to reload them. Let me know if you have any questions, thanks!

Best wishes,
HTseng

**Comment by MinLiAmoy** [comment]
Dear HTseng,

I successfully load the pre-trained model you provided.
In order to fully use GPU memory, I set batch size to be 128 before, and the loss did not converge. Then I changed the batch size to 40 (default), then it works.

I also added a learning scheduler: torch.optim.lr_scheduler.ReduceLROnPlateau(), and the accuracy of ResNetCRNN on testing data increased to 90.42%. It only needs 3 more lines of code, and I guess you can use it in your code :)

Best regards,
Min

**Comment by HHTseng** [comment]
Dear Min,
Thank you for letting me know the large batch, 128, does not work out directly, it's interesting to know.  Also your idea adding a learning scheduler is brilliant; I did not know it would increase the accuracy that much, it will be incorporated into the code as soon as possible. Really appreciate your nice techniques and useful information!! 

Best wishes,
HTseng
